packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -563,6 +570,21 @@ class HotRunner extends ResidentRunner {
     );
   }
 
+  bool _isPaused() {
+    for (FlutterDevice device in flutterDevices) {
+      for (FlutterView view in device.views) {
+        if (view.uiIsolate != null) {
+          final ServiceEvent pauseEvent = view.uiIsolate.pauseEvent;
+          if (pauseEvent != null && pauseEvent.isPauseEvent) {
```

Comment:
Yuck....

Can you add a DartDoc comment to `vmservice.dart` so that gets capture on the flutter tools side as well?